### ISSUE TITLE
Accessibility - Contrast ratio needs changing on disclaimer

### ISSUE THREAD

**Issue by DimplePatel** [MHRA/products#135]
**Describe the bug**
Contrast ratio for the hyperlink on the disclaimer doesn't meet AA standards

**To Reproduce**
1) Open the mhra website as a first visit
2) Perform a search so you trigger the disclaimer message
3) The hyperlink blue colour only has a contrast ratio of 4.26:1

**Expected behavior**
The colour now being used for the cookie policy in the cookie banner should be used here 

**Screenshots**
![Screenshot 2019-12-11 at 18 38 40](https://user-images.githubusercontent.com/6122821/70650366-db414e00-1c46-11ea-8d05-585565b8b18b.png)

**Additional context**
original fix was here https://github.com/MHRA/products/pull/131

**Comment by DimplePatel** [comment]
The link for yellow card points to 'anchor color' which is #1d70b8 . This has a contrast ratio of below 4.5:1

**Comment by DimplePatel** [comment]
<img width="869" alt="Screenshot 2019-12-12 at 17 10 13" src="https://user-images.githubusercontent.com/6122821/70733518-67fd1200-1d02-11ea-8c0a-c085910b9148.png">
